content/applications/hr/time_off.rst

```diff
@@ -449,8 +502,11 @@ Enter the following information on the form:
   Navigate to the file(s) to attach, then click the :guilabel:`Open` button. The files then appear
   on the time off request form. Multiple documents can be attached, if necessary.
 
-When the form is complete, click the :guilabel:`Save` button to save the information, and submit the
-request.
+If the request was created from the :guilabel:`dashboard`, click the :guilabel:`Save & Close` button
```

Comment:
Where can I find this dashboard? You can add menuselection instructions to show the reader how to get there, or provide a ref link to jump there so the reader can learn more about this dashboard (: